examples/stocks/lib/main.dart

```diff
@@ -94,6 +94,13 @@ class StocksAppState extends State<StocksApp> {
     return null;
   }
 
+  Future<LocaleQueryData> _onLocaleChanged(ui.Locale locale) {
+    return initializeMessages(locale.toString()).then((_) {
+      Intl.defaultLocale = locale.toString();
+      return StockStrings.instance;
```

Comment:
Should we just inline the Intl calls?

Maybe the object we should get back from Locale.of() is actually an Intl object for the current locale, and rather than using Intl static calls in a static StockStrings class, we do everything through that object, and just define the strings inline where we use them.